src/data/calculatorPages.ts

```diff
@@ -285,20 +284,32 @@ const data: Pages = {
       },
       {
         type: BodyType.LINK,
-        text: "https://www.courts.wa.gov/forms/?fa=forms.contribute&formID=38",
+        text: "CrRLJ 09.0300",
         href: "https://www.courts.wa.gov/forms/?fa=forms.contribute&formID=38",
       },
+      {
+        type: BodyType.PARAGRAPH,
+        text: "To better understand misdemeanor conviction vacation, please click the link below:"
+      },
+      {
+        type: BodyType.LINK,
+        text: "Click here.",
+        href: "https://www.washingtonlawhelp.org/resource/get-misdemeanor-convictions-off-your-criminal-record"
+      }
     ],
     disclaimer,
   },
   "mar-ineligible-1": {
     header:
       "As of now, it seems that you may NOT be eligible to vacate your marijuana misdemeanor conviction because you have not completed the terms of your offense.",
     disclaimer,
+  },
+  "mar-ineligible-2": {
+
   },
   "mar-to-main": {
     header:
-      "Based on your answers, it looks like you don't qualify for the marijuana misdemeanor exception.",
+      "Based on your response, it looks like you don't qualify for the marijuana misdemeanor exception.",
```

Comment:
The text for the button on this page is "Continue Journey." Although for consistency it might make more sense to not capitalize journey based on other flagged changes.

Reply:
Agreed. I'll make it "Continue journey".